wic/iot2050-swu-secure.wks.in

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) Siemens AG, 2021-2022
+#
+# Authors:
+#  Jan Kiszka <jan.kiszka@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+include ebg-signed-bootloader.inc
+
+# EFI Boot Guard environment/config partitions plus Kernel files
+part --source efibootguard-boot --size 32M --extra-space 0 --overhead-factor 1 --label BOOT0 --align 1024 --part-type=0700 --sourceparams "revision=2,signwith=/usr/bin/sign_secure_image.sh"
+part --source efibootguard-boot --size 32M --extra-space 0 --overhead-factor 1 --label BOOT1 --align 1024 --part-type=0700 --sourceparams "revision=1,signwith=/usr/bin/sign_secure_image.sh"
```

Comment:
Why these boot partitions are not part of the included `ebg-signed-bootloader.inc`?

I notice that the boot partitions are part of the `ebg-sysparts.inc` for non-secure configuration.

Reply:
Good point, I guess I missed that imbalance while refactoring isar-cip-core. Let me quickly clean this up: https://gitlab.com/cip-project/cip-core/isar-cip-core/-/commit/71c708b27d2a0a6788639f3badeb5c13c20ff22f. Will be picked up here once merged.